src/intTest/org/onflow/flow/sdk/ExposeAccountKeyIssueTest.kt

```diff
@@ -101,8 +109,8 @@ class ExposeAccountKeyIssueTest {
 
         val signatureAlgorithm2 = SignatureAlgorithm.ECDSA_P256
         val hashAlgorithm2 = HashAlgorithm.SHA3_256
-        val pair2 = Crypto.generateKeyPair(signatureAlgorithm1)
-        val signer2 = Crypto.getSigner(pair1.private, hashAlgorithm1)
+        val pair2 = Crypto.generateKeyPair(signatureAlgorithm2)
+        val signer2 = Crypto.getSigner(pair2.private, hashAlgorithm2)
```

Comment:
Was the use of the version 1 variables in the original code an error?

Reply:
Yes, I noticed that the second part of the test was re-using pair1 signer and hasher instead of setting a new assertion with pair2